pkgs/development/compilers/scryer-prolog/default.nix

```diff
@@ -13,30 +13,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "scryer-prolog";
-  version = "0.9.0";
+  version = "0.9.1";
 
   src = fetchFromGitHub {
     owner = "mthom";
     repo = "scryer-prolog";
     rev = "v${version}";
-    sha256 = "3NHpEg6QaUaqbBCq8uM5hFcqS24q4XrOnKjMmn8Z1Dg=";
+    sha256 = "bDLVOXX9nv6Guu5czRFkviJf7dBiaqt5O8SLUJlcBZo=";
   };
 
   cargoPatches = [
     # Use system openssl, gmp, mpc and mpfr.
     ./cargo.patch
-
-    ./fix-tests.patch
-
-    # Avoid testing failing with "couldn't save history"
-    (fetchpatch {
-      name = "fix-tests-1";
-      url = "https://patch-diff.githubusercontent.com/raw/mthom/scryer-prolog/pull/1342.patch";
-      sha256 = "2N0AOkFuf+H/aUn2QTXgmqjmvShTxHxB6kNuNdNoVRI=";
-    })
   ];
 
-  cargoSha256 = "nqAHVXAmTW9mdE2L2yhpOTz16JbYgQUmCgiFq9pBzUU=";
+  cargoSha256 = "A6HtvxGTjJliDMUSGkQKB13FRyfBU4EPvrlZ97ic0Ic=";
 
   nativeBuildInputs = [ pkg-config rustfmt];
   buildInputs = [ openssl gmp libmpc mpfr ];
```

Comment:
openssl must be remove here.

Reply:
There is still an indirect dependency on openssl through the native-tls lib.

Reply:
You are right, sorry. I saw [this](https://github.com/mthom/scryer-prolog/commit/9c1de8e00bcf8cf110066fc2a5aff93625fdf9d1) commit and thought that the openssl dependency was completely removed.